docs/folder-tree.md

```diff
@@ -0,0 +1,366 @@
+    |   |   |   └── styles.ts
+    |   |   ├── Minimizer/
+    |   |   |   ├── index.tsx
+    |   |   |   ├── Minimizer.tsx
+    |   |   |   └── styles.ts
+    |   |   ├── RegionDetails/
+    |   |   |   ├── CollapsibleContent.tsx
+    |   |   |   ├── ComparisonSection.tsx
+    |   |   |   ├── DataSection.tsx
+    |   |   |   ├── index.tsx
+    |   |   |   ├── RegionDetails.tsx
+    |   |   |   └── styles.ts
+    |   |   ├── tests/
+    |   |   |   ├── mock.ts
+    |   |   |   └── Sidebar.spec.tsx
+    |   |   ├── index.tsx
+    |   |   ├── SideBar.tsx
+    |   |   └── Modal.tsx
+    |   └── Theme/
+    |       ├── index.tsx
+    |          └── palette.ts
+    ├── config/
+    |   ├── demographic.ts
+    |   ├── economic.ts
+    |   ├── index.ts
+    |   ├── social.ts
+    |   └── socioeconomic.ts
+    ├── context/
+    |   ├── district/
+    |   |   ├── highlightedContext.tsx
+    |   |   └── selectedContext.tsx
+    |   ├── state/
+    |   |   ├── highlightedContext.tsx
+    |   |   └── selectedContext.tsx
+    |   ├── comparisonContext.tsx
+    |   └── sidebarContext.tsx
+    ├── customTypes/
+    |   ├── district.ts
+    |   ├── map.ts
+    |   └── state.ts
+    ├── data/
+    |   ├── states/
+    |   |   ├── AC_Municipios_2020.json
+    |   |   ├── AL_Municipios_2020.json
+    |   |   ├── AM_Municipios_2020.json
+    |   |   ├── AP_Municipios_2020.json
+    |   |   ├── BA_Municipios_2020.json
+    |   |   ├── CE_Municipios_2020.json
+    |   |   ├── DF_Municipios_2020.json
+    |   |   ├── ES_Municipios_2020.json
+    |   |   ├── GO_Municipios_2020.json
+    |   |   ├── index.ts
+    |   |   ├── MA_Municipios_2020.json
+    |   |   ├── MG_Municipios_2020.json
+    |   |   ├── MS_Municipios_2020.json
+    |   |   ├── MT_Municipios_2020.json
+    |   |   ├── PA_Municipios_2020.json
+    |   |   ├── PB_Municipios_2020.json
+    |   |   ├── PI_Municipios_2020.json
+    |   |   ├── PR_Municipios_2020.json
+    |   |   ├── RJ_Municipios_2020.json
+    |   |   ├── RN_Municipios_2020.json
+    |   |   ├── RR_Municipios_2020.json
+    |   |   ├── RS_Municipios_2020.json
+    |   |   ├── SC_Municipios_2020.json
+    |   |   ├── SE_Municipios_2020.json
+    |   |   ├── SP_Municipios_2020.json
+    |   |   └── TO_Municipios_2020.json
+    |   ├── BR_UF_2020.json
+    |   ├── Data.json
+    |   ├── ExtremePopValueByState.json
+    |   ├── GeoSes.json
+    |   └── StateData.json
+    ├── hook/
+    |   ├── useFeature.ts
+    |   └── useMap.ts
+    ├── pages/Main/
+    |   ├── hook/
+    |   |   ├──  index.ts   
+    |   |   └── useMain.tsx 
+    |   ├── index.tsx
+    |   ├── Main.tsx
+    |   └── styles.ts
+    ├── utils/
+    |   ├── coloManipulator.ts
+    |   └──formatValue.ts
+    ├── index.tsx
+    └──main.tsx 
+    
+ ``` 
+ 
+## About folders of project
+
+    src/
+
+The src folder contains the structure of the codes and the project divided in many smaller parts for a better understandment of the structures that make up the Atlas.
```

Comment:
**suggestion:** for all the folder here, make the path in bold mode and after this add the explanation
```suggestion
    `src/`: The src folder contains the structure of the codes and the project divided in many smaller parts for a better understandment of the structures that make up the Atlas.
```